libs/ui/src/Loading.test.tsx

```diff
@@ -3,16 +3,18 @@ import { render } from '@testing-library/react'
 
 import { Loading } from './Loading'
 
-it('renders default', () => {
-  const { container } = render(<Loading />)
-  expect(container.firstChild).toMatchSnapshot()
-})
+describe('Renders Loading', () => {
```

Comment:
Is this `describe` useful?

Reply:
Am I using it correctly? To wrap a set of tests with the "theme"? Happy to update pattern if you've got one to suggest.

Reply:
Yeah that is why you'd use it, but typically you'd use it to differentiate from some _other_ `describe`. There don't seem to be any others in this file.